src/config/rcxml.c

```diff
@@ -895,9 +895,13 @@ entry(xmlNode *node, char *nodename, char *content)
 	} else if (!strcasecmp(nodename, "outlines.windowSwitcher")) {
 		set_bool(content, &rc.window_switcher.outlines);
 	} else if (!strcasecmp(nodename, "allWorkspaces.windowSwitcher")) {
+		set_bool(content, &rc.window_switcher.allworkspaces);
 		if (parse_bool(content, -1) == true) {
 			rc.window_switcher.criteria &=
 				~LAB_VIEW_CRITERIA_CURRENT_WORKSPACE;
```

Comment:
No need to parse the bool twice, suggest instead:
```c
if (rc.window_switcher.allworkspaces) {
	[..]
}
```

`allworkspaces` is weird to type and read IMHO, maybe `all_workspaces` for the new config var would be better?

Reply:
all_workspaces is fine, so is workspaces_all